docs/docs/modeling.md

```diff
@@ -57,6 +57,7 @@ classDiagram
         TenantId tenantId
         String email
         String name
+        Number ExtraTaskCount
```

Comment:
This is a very small thing, but `Number` could be either an integer or a floating number. In this case, I think the `Number` should be replaced with `Int` (Integer).